huggle/Localization/en.xml

```diff
@@ -260,7 +260,7 @@
   <string name="login-progress-user">Checking user configuration for $1</string>
   <string name="login-progress-user-info">Checking user info for $1</string>
   <string name="login-progress-whitelist">Retrieving user whitelist...</string>
-  <string name="login-projets">Projects</string>
+  <string name="login-projets">Projets</string>
```

Comment:
but it is PROJECTS I said you made a typo in "login-projets" and now you have the same typo in both strings